### PR TITLE
OSDOCS-2386: MCO needs to support RHEL 7 and 8 nodes in OCP 4.9

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-2386

### CODE REVIEW

**Comment by mburke5678** [comment]
@jkyros Can you please take a look at these release note changes related to https://issues.redhat.com/browse/GRPA-3517

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -21,12 +21,7 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 
 {product-title} {product-version} is supported on {op-system-base-full} 7.9 or later, as well as on {op-system-first} 4.8.
 
-You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base-full} 7.9 or later for compute machines.
```
Comment (jkyros) [inline release_notes/ocp-4-9-release-notes.adoc:24]: I'd like to keep this line -- the control plane still has to remain RHCOS, but the compute can be RHEL7.9+

**Comment by mburke5678** [comment]
[QE review in dev jira issue](https://issues.redhat.com/browse/GRPA-3517?focusedCommentId=18837775&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-18837775).

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -94,6 +90,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-9-machine-api"]
 === Machine API
 
+[id="ocp-4-9-machine-api-rhel8"]
+==== {op-system-base-full} 8 now supported for compute machines
+Starting in {product-title} {product-version}, you can now use {op-system-base-full} 8 for compute machines.  Previously, RHEL 8 was not supported for compute machines.
```
Comment (adellape) [inline release_notes/ocp-4-9-release-notes.adoc:95]: Extra space between sentences.

**Review by adellape — COMMENTED** [review]
:+1:

**Pull request merged**